Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/MenuFragment.java

```diff
@@ -137,6 +119,37 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
     }
   }
 
+  private void setInsets()
+  {
+    ViewCompat.setOnApplyWindowInsetsListener(mBinding.getRoot(), (v, windowInsets) ->
+    {
+      Insets cutInsets = windowInsets.getInsets(WindowInsetsCompat.Type.displayCutout());
+      int left = 0;
+      int right = 0;
+      if (ViewCompat.getLayoutDirection(v) == ViewCompat.LAYOUT_DIRECTION_LTR)
+      {
+        left = cutInsets.left;
+      }
+      else
+      {
+        right = cutInsets.right;
+      }
+
+      // Don't use padding if the navigation bar isn't in the way
+      if (getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE)
```

Comment:
On tablets (and some other form factors too), there is a navigation bar at the bottom even in landscape.

Reply:
Ah shoot, I'll try reusing your previous method for finding the navigation bar padding then. I don't think there's a way to get the side of the screen where the navigation bar exists besides that.